Add TFDS Python Interop Demo

WilliamHYZhang opened the pull request:
Added a simple demo on how to load datasets in Swift with TFDS.

rickwierenga commented on TFDS_Python_Interoperability.ipynb:
```diff
@@ -0,0 +1,342 @@
+            "tags": []
+          }
+        }
+      ]
```
Consider placing these in one figure with `plt.subplot(1,3,i)`.

rickwierenga commented on TFDS_Python_Interoperability.ipynb:
```diff
@@ -0,0 +1,342 @@
+        "print(ds_train)"
+      ],
+      "execution_count": 15,
+      "outputs": [
```
Please consider rerunning the notebook so all the numbers are sequential. This is a very nit-picky detail, but I think it's best to keep this consistent.

WilliamHYZhang commented:
@rickwierenga sounds good :+1:

I'm currently trying to see if we can use the "Conversion with `numpy.ndarray`" specified [here](https://www.tensorflow.org/swift/tutorials/python_interoperability#conversion_with_numpyndarray), do you think if we loaded with `as_numpy` it could work?

rickwierenga commented:
Yes, I think that would be the best way to go about this since tf tensors can't be converted directly to Swift tensors yet.

WilliamHYZhang commented:
@rickwierenga should be good, added conversion to Swift `Tensor` as well as addressed your comments.

UPDATE: also added example to convert the entire dataset split if desired :)

rickwierenga commented on TFDS_Python_Interoperability.ipynb:
```diff
@@ -0,0 +1,310 @@
+        "%include \"EnableIPythonDisplay.swift\"\n",
+        "IPythonDisplay.shell.enable_matplotlib(\"inline\")"
+      ],
+      "execution_count": 0,
```
I think these got changed at some point. You can easily change them by editing the raw file.

rickwierenga replied:
Just to be clear, I only meant line 57.

rickwierenga commented on TFDS_Python_Interoperability.ipynb:
```diff
@@ -0,0 +1,310 @@
+        "let tfds = Python.import(\"tensorflow_datasets\")\n",
+        "let tf = Python.import(\"tensorflow\")\n",
+        "\n",
+        "%include \"EnableIPythonDisplay.swift\"\n",
+        "IPythonDisplay.shell.enable_matplotlib(\"inline\")"
```
These lines should be in their own cell because of [this](https://bugs.swift.org/browse/TF-183). See [this guide](https://www.tensorflow.org/swift/tutorials/python_interoperability#displaying_images).

WilliamHYZhang commented:
@rickwierenga thanks for the suggestions, pushed the new file.
